### PR TITLE
Dacer

### PR DESCRIPTION
**Pull request by wadaniel**

### CODE REVIEW

**Comment by wadaniel** [comment]
added single upswing example,
resolved segfault in acer
added safe exp sum in dqn

File: examples/learning/reinforcement/double_pendulum/README.rst [hunk 1]
```diff
@@ -1,4 +0,0 @@
-Double Pendulum (Python)
-========================
-
-Examples of different algorithms solving the double pendulum on a cart swing-up problem implemented in python.
```
Comment (webepasc) [inline examples/learning/reinforcement/double_pendulum/README.rst:4]: Why no README anymore?

File: source/modules/solver/agent/discrete/dACER/dACER._cpp [hunk 2]
```diff
@@ -124,7 +124,8 @@ void dACER::trainAgent()
   std::shuffle(trajectoryIndexes.begin(), trajectoryIndexes.end(), *mt);
 
   // Performing off-policy updates
-  for (size_t i = 0; i < _offPolicyUpdates; i++)
+  size_t numOffPolicyUpdates = trajectoryIndexes.size() < _offPolicyUpdates ? trajectoryIndexes.size() : _offPolicyUpdates;
```
Comment (webepasc) [inline source/modules/solver/agent/discrete/dACER/dACER._cpp:127]: Why?

**Review by webepasc — DISMISSED** [review]
GTG (good to go)

**Review by SergioMartin86 — APPROVED** [review]

**Pull request merged**